tests/FlatBuffers.Test.Swift/Tests/FlatBuffers.Test.SwiftTests/monster_test_generated.swift

```diff
@@ -50,6 +50,14 @@ public enum Any_: UInt8, Enum {
 	public static var min: Any_ { return .none }
 }
 
+struct Any_Union {
+	var type: Any_
+	var value: FlatBuffersObjectAPI?
```

Comment:
This is a bit of a weird name for the super type of all object API classes? Maybe `NativeTable`, `TableT`, or something else? `Object` if its namespaced?

Reply:
So replacing the `FlatBuffersObjectAPI` with `NativeTable`?

Reply:
Yes, or any other option. Just putting `FlatBuffers` in there doesn't make sense since this all FlatBuffers, and `API` is too meta. Actually maybe `NativeObject` is better, since it can be other things than a table?